project-base/storefront/components/Forms/Textarea/TextareaControlled.tsx

```diff
@@ -18,9 +18,17 @@ export const TextareaControlled: FC<TextareaControlledProps> = ({ name, render,
     } = useController({ name, control });
     const textareaId = formName + '-' + name;
 
+    const combinedOnBlurHandler: FocusEventHandler<HTMLTextAreaElement> = (event) => {
```

Comment:
What about more common naming such as `handleOnBlur`?

Reply:
I would like to keep this naming of "combined" where they really are "combined", meaning where we combine two onEvent handlers to one. This is used where you want to provide both your own callback and the react hook form callback. It's been like that for quite some time and it still makes sense to me.

Reply:
Really? I can't find any other place where we would use it. I still consider this as confusing because we use approach with custom function such as `onBlurHandler` and inside you do whatever magic you want (including calling another callback).

Reply:
There are 4 palces where we use it:

<img width="607" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/5d5479f5-1e99-4ab8-916c-34da9094b8be">

You have to combine the handlers somewhere. Would you rather do it inside the component, or everywhere you use it? Or are you only saying that you do not like the name? We can discuss it on a call if you want.

Reply:
As a developer using the component, I do not care about how the handlers are combined, I only care about my handler being called. However, If I did this:

<img width="874" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/640744c6-d82f-4ec8-93b6-5cc3b44aedc1">

Then the provided `onBlur` would be ignored. If I change the order of the props:

<img width="874" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/550e2cac-7572-4f88-81b6-866e200afe8c">


Then react hook form callbacks would not work. because of that, we need to combine the handlers into a single, _combined_ handler. Which is my approach

Reply:
I was wrong, this was actually introduced here. I renamed it as you wanted...